bpdv3/Assets/Scripts/Managers/TileController.cs

```diff
@@ -54,12 +65,23 @@ private void Update()
                 var tileMovement = _directionOfMovement.normalized * (Time.deltaTime * (1.25f / 3f));
```

Comment:
Do the numbers as variables!

Reply:
I see, such as constants. Thanks for catching this. Will do.